src/main/java/uk/gov/digital/ho/pttg/api/AuditHistoryResource.java

```diff
@@ -54,6 +54,18 @@ public AuditHistoryResource(AuditHistoryService auditHistoryService, RequestData
         return result;
     }
 
+    @GetMapping(value = "correlationids", produces = APPLICATION_JSON_VALUE)
```

Comment:
Should the mapping "value" have a leading slash?

Reply:
Yes so I've changed it. Interesting how it works without though!